### PR TITLE
refactor: Replaced GCS sources

### PR DESCRIPTION
**Pull request by Anush008**
## Description

This is Part I of adding support for quantized models and lazy loading required files.

Part II - #200

### CODE REVIEW

**Comment by Ya-shh** [comment]
@Anush008,
When integrating support for quantized models through the Hugging Face source, I have encountered the need to experiment with multiple models. While testing various ONNX models, I've found that only a small subset of them have accurate ONNX weights.

**Comment by Ya-shh** [comment]
I see tests are failing for `bge-base-en`. I just it converted to onnx & It passed all the tests locally. You can use "yashvardhan7/bge-base-en-onnx" instead .
https://huggingface.co/yashvardhan7/bge-base-en-onnx/tree/main

File: fastembed/text/onnx_embedding.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
         "description": "Base English model",
         "size_in_GB": 0.42,
         "sources": {
-            "url": "https://storage.googleapis.com/qdrant-fastembed/fast-bge-base-en.tar.gz",
+            "hf": "aident-ai/bge-base-en-onnx",
```
Comment (Ya-shh) [inline fastembed/text/onnx_embedding.py:17]: you can use `yashvardhan7/bge-base-en-onnx` here

Comment (Anush008) [reply fastembed/text/onnx_embedding.py:17]: Thanks Yash. We're probably will proceed with #201 instead.

Comment (Anush008) [reply fastembed/text/onnx_embedding.py]: Committed your source. Just in case.

**Comment by joein** [comment]
closing in favour of #201, feel free to re-open if I got it wrong

**Pull request closed without merging**